tasks/swagger-js-codegen.js

```diff
@@ -33,7 +45,8 @@ module.exports = function (grunt) {
                     if(err) {
                         deferred.reject(err);
                     } else {
-                        generateApi(api, data, dest, fname);
+                        var swagger = JSON.parse(data);
```

Comment:
I was thinking creating function to replace fs.readFile bit but I was too lazy..

Could this work?
```javascript
fs.readFile(api.swagger, 'UTF-8', function(err, data){
    if(err) {
        deferred.reject(err);
    } else {
        var swagger;
        if(testIfYamlExtension(api.swagger)) {
            swagger = yaml.parse(data);
        } else {
            swagger = JSON.parse(data);
        }
        
        generateApi(api, swagger, dest, fname);
        deferred.resolve();
    }
});
```